packages/react-native-client/src/components/FishjamSandboxRoom.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from 'react';
```

Comment:
I'm not sure if this should be indeed part of SDK 🤔 
I know this would simplify examples, but it would also move part of example app into SDK. And this code can't be used in production app anyway.

Reply:
Maybe such component can be part of example app?

Reply:
I agree, it might not fit perfectly, but it's not so bad to have this if it simplifies it for our users. If we add it to the example it won't be possible to import it so it defeats the purpose. The key here is to be able to easily test it inside your app.